packages/slice-machine/src/domain/customType.ts

```diff
@@ -129,6 +129,28 @@ export function getSectionSliceZoneConfig(
   return maybeSliceZone?.config ?? undefined;
 }
 
+// TODO add tests
+// We block user from deleteing section containing UID field,
+// so we can expect uid to be in the main section
+export function getUidField(customType: CustomType): UID | undefined {
```

Comment:
💡#nitpick

I see that sometimes you are doing `Uid` or `UID` for example for `UIDEditor`.
I would be consistent and do `getMainSectionUIDField `.

Reply:
done